sdcm/sct_runner.py

```diff
@@ -256,6 +257,12 @@ def _create_instance(self,
                          test_duration: Optional[int] = None) -> Any:
         ...
 
+    @abstractmethod
+    def set_tags(self,
+                 sct_runner_info: SctRunnerInfo,
+                 tags: dict):
+        ...
```

Comment:
why this implementation is empty ?, all the implementations below look identical...
or it's not yet tested on the 3 providers ?

Reply:
This was likely the abstract method or just a placeholder. Either way the method has implementations in each concrete child class now.